YAMP.Core/Help/Documentation.cs

```diff
@@ -307,7 +307,9 @@ static String GetLocalized(String key)
         {
             var value = default(String);
             var dictionary = Localization.Current ?? Localization.Default;
-            dictionary.TryGetValue(key, out value);
+            //Fallback to show something while key/descriptions are being replaced
+            if (!dictionary.TryGetValue(key, out value))
+                value = key;
```

Comment:
You're right - I've added that method with explicit checking. The second argument I had in my mind for that line was to "force" implementations to properly specify a key / add it to the dictionary. Right now, however, this "adding" construct (which needs to be done on plugin initialization) is not ready.

So yeah, let's go for that change and let's print the keys as last resort!